substrate/frame/election-provider-support/benchmarking/src/inner.rs

```diff
@@ -20,17 +20,19 @@
 
 use alloc::vec::Vec;
 use codec::Decode;
-use frame_benchmarking::v1::benchmarks;
+use frame_benchmarking::v2::*;
 use frame_election_provider_support::{NposSolver, PhragMMS, SequentialPhragmen};
-
-pub struct Pallet<T: Config>(frame_system::Pallet<T>);
-pub trait Config: frame_system::Config {}
+use sp_runtime::Perbill;
 
 const VOTERS: [u32; 2] = [1_000, 2_000];
 const TARGETS: [u32; 2] = [500, 1_000];
 const VOTES_PER_VOTER: [u32; 2] = [5, 16];
-
 const SEED: u32 = 999;
+
+pub trait Config: frame_system::Config {}
+
+pub struct Pallet<T: Config>(frame_system::Pallet<T>);
```

Comment:
why are they necessary?

Reply:
These come from the old file.

Reply:
Ah my bad, I didn't see the were written above before.